airbyte-integrations/connectors/source-amazon-ads/source_amazon_ads/schemas/common.py

```diff
@@ -42,6 +43,7 @@ class MetricsReport(CatalogModel):
     profileId: int
     recordType: str
     reportDate: str
+    updatedAt: datetime
```

Comment:
yes, this field `updatedAt` added for all 4 streams: `sponsored_display_report_stream`, `sponsored_products_report_stream`, `sponsored_brands_report_stream`, `sponsored_brands_video_report_stream`.

But only for `sponsored_products_report_stream` it's added to primary_key

I updated primary_key only for `sponsored_products_report_stream` because rest 3 streams don't have data for now and I cannot test them